package/Config.in

```diff
@@ -952,6 +955,7 @@ menu "External python modules"
 	source "package/python-oauthlib/Config.in"
 	source "package/python-paho-mqtt/Config.in"
 	source "package/python-pam/Config.in"
+	source "package/python-pantilthat/Config.in"
```

Comment:
This should have been added here, indeed, but I think it's missing the package folder.